pkg/operator/controller/ingress_status.go

```diff
@@ -205,3 +209,80 @@ func getEventsByReason(events []corev1.Event, component, reason string) []corev1
 	}
 	return filtered
 }
+
+func computeDNSStatus(ic *operatorv1.IngressController, wildcardRecord *iov1.DNSRecord, dnsConfig *configv1.DNS) []operatorv1.OperatorCondition {
+	if dnsConfig.Spec.PublicZone == nil && dnsConfig.Spec.PrivateZone == nil {
+		return []operatorv1.OperatorCondition{
+			operatorv1.OperatorCondition{
+				Type:    operatorv1.DNSManagedIngressConditionType,
```

Comment:
In the e2e test, if this condition is true, wait for DNS on the ingress status to be ready